src/eventcontroller/EventController.ts

```diff
@@ -1,16 +1,34 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import EventObserver from '../eventobserver/EventObserver';
+import EventReporter from '../eventreporter/EventReporter';
 import AudioVideoEventAttributes from './AudioVideoEventAttributes';
 import DeviceEventAttributes from './DeviceEventAttributes';
 import EventName from './EventName';
 import MeetingHistoryState from './MeetingHistoryState';
 
 /**
- * [[EventController]] keeps track of events and notifies audio-video observers.
- * An event describes the success and failure conditions for the meeting session.
+ * [[EventController]] keeps track of a list of event observers and notifies them of SDK events.
+ * An event can describe success or failure of SDK components that make use of this controller.
+ * Example: The success and failure for a meeting session.
+ * If EventIngestionUrl is available in MeetingSessionConfiguration, then the generated meeting events will
```

Comment:
nit: Would be great to backtick these coding names. EventIngestionUrl, MeetingSessionConfiguration and so on. Can skip if not followed elsewhere to get this in in-time.

Reply:
I didn't know you could in docstring, updated